ujit_core.h

```diff
@@ -61,12 +64,12 @@ typedef struct BlockId
 static const blockid_t BLOCKID_NULL = { 0, 0 };
 
 /// Branch code shape enumeration
-enum uint8_t
+typedef enum : int8_t
```

Comment:
```suggestion
typedef enum branch_shape
```

Sorry for suggesting this, this is not a C feature.